cli/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "star-frame"
```

Comment:
The name of the package should follow the format of `star_frame_*` such as `star_frame_cli`. This can't be published with this name because it will conflict with `star_frame`. You can declare the binary name separately